types/vara/index.d.ts

```diff
@@ -0,0 +1,155 @@
+// Type definitions for vara 1.1
+// Project: https://github.com/akzhy/vara#readme
+// Definitions by: Richard Haddad <https://github.com/Chnapy>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Comments are from the documentation: http://vara.akzhy.com/documentation/
+ */
+
+/**
+ * The options given below will be applicable to every text created,
+ * however they will not override the options set above.
+ * They will work as secondary options.
+ */
+export interface TextProperties {
+    /**
+     * Size of the text
+     */
+    fontSize?: number;
+    /**
+     * Width / Thickness of the stroke
+     */
+    strokeWidth?: number;
+    /**
+     * Color of the text
+     */
+    color?: string;
+    /**
+     * Duration of the animation in milliseconds
+     */
+    duration?: number;
+    /**
+     * Text align, accepted values are left,center,right
+     */
+    textAlign?: 'left' | 'center' | 'right';
+    /**
+     * Whether to animate the text automatically
+     */
+    autoAnimation?: boolean;
+    /**
+     * Whether the animation should be in a queue
+     */
+    queued?: boolean;
+    /**
+     * Space between each character
+     */
+    letterSpacing?: number;
+}
+
+export interface TextStep extends TextProperties {
+    /**
+     * Text to be shown
+     */
+    text: string;
+    /**
+     * String or integer, for if animations are called manually or when using the get() method.
+     * Default is the index of the object.
+     */
+    id?: string | number;
+    /**
+     * x coordinate of the text
+     */
+    x?: number;
+    /**
+     * y coordinate of the text
+     */
+    y?: number;
+    /**
+     * Whether the x or y coordinate should be from its calculated position,
+     * ie the position if x or y coordinates were not applied
+     */
+    fromCurrentPosition?: {
+        x?: boolean;
+        y?: boolean;
+    };
+    /**
+     * Delay before the animation starts in milliseconds
+     */
+    delay?: number;
+}
+
+export interface TextElements {
+    /**
+     * Array of svg g elements, each representing a letter
+     */
+    characters: SVGGElement[];
+
+    /**
+     * Svg g wrapping the text block
+     */
+    container: SVGGElement;
+}
+
+export default class Vara {
```

Comment:
It's not `export default`: https://github.com/akzhy/Vara/blob/master/src/vara.js#L512

Reply:
Ok my bad.
I don't know the equivalent of `module.exports = Vara;` in ES6 syntax.